packages/helix_editor.rb

```diff
@@ -0,0 +1,53 @@
+require 'package'
+
+class Helix_editor < Package                 # The first character of the class name must be upper case
+  description 'A vim inspired editor with LSP support'
+  homepage 'https://helix-editor.com/'
+  version '22.12'
+  license 'MPL-2.0' # license of source
+  compatibility 'all'
+  source_url 'https://github.com/helix-editor/helix/archive/refs/tags/22.12.tar.gz'
```

Comment:
A shortened version of the url, `https://github.com/helix-editor/helix/archive/22.12.tar.gz` should work too.

Reply:
I created one (see comment in code) but I don't know why: "error: could not find `Cargo.toml` in `/usr/local/tmp/crew/helix_editor.20230216163702.dir` or any parent directory".
By the way, what is the motive of using a shortened path here?

Reply:
Shorter is better?  No reason other than saving characters. :)

Reply:
You may need to `FileUtils.touch 'Cargo.toml'` somewhere.

Reply:
Maybe but why does it work with the original link? I don't get it.

Reply:
I removed the shortened link. The original link is more understandable and direct...